webpack.config.babel.js

```diff
@@ -89,6 +90,24 @@ module.exports = {
 	],
 
 	plugins: ([
+		new V8LazyParseWebpackPlugin(),
+		new webpack.optimize.UglifyJsPlugin({
```

Comment:
This is actually going to run Uglify in development mode, and twice in production mode.
We run `webpack -p` for production, which include UglifyJSPlugin, DedupePlugin by default.